examples/inverse/evoked_ers_source_power.py

```diff
@@ -70,17 +71,32 @@
 # will be correctly preserved.
 
 rank = mne.compute_rank(epochs, tol=1e-6, tol_kind='relative')
-active_win = (0.5, 1.5)
-baseline_win = (-1, 0)
-baseline_cov = compute_covariance(epochs, tmin=baseline_win[0],
-                                  tmax=baseline_win[1], method='shrunk',
+win_active = (0.5, 1.5)
+win_baseline = (-1, 0)
+cov_baseline = compute_covariance(epochs, tmin=win_baseline[0],
+                                  tmax=win_baseline[1], method='shrunk',
                                   rank=rank, verbose=True)
-active_cov = compute_covariance(epochs, tmin=active_win[0], tmax=active_win[1],
+cov_active = compute_covariance(epochs, tmin=win_active[0], tmax=win_active[1],
                                 method='shrunk', rank=rank, verbose=True)
 
-# Weighted averaging is already in the addition of covariance objects.
-common_cov = baseline_cov + active_cov
-mne.viz.plot_cov(baseline_cov, epochs.info)
+# Weighted averaging is already in the addition of covariance objects
```

Comment:
Not sure I fully understand this comment. Do you mean that weighted averaging is taken care of?

Reply:
That line just got moved, I didn't change it in any way. I didn't write it so I'm not sure the answer, I would infer from the authors that Marijn wrote it